lesson07.blog-api-server/static_server2.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+const http = require("http");
+const fs = require("fs");
+const url = require("url");
+
+const contentTypes = new Map();
+contentTypes.set("html", "text/html");
+contentTypes.set("js", "text/javascript");
+contentTypes.set("css", "text/css");
+contentTypes.set("json", "application/json");
+contentTypes.set("png", "image/png");
+
+http
+  .createServer(function(req, res) {
+    const reqUrl = url.parse(req.url, true);
+    const ext = reqUrl.pathname.split(".")[1];
```

Comment:
what will happen if `pathname` value is `/index.`?
what will happen if `pathname` value is `/`?